module/VuFind/src/VuFind/Db/Table/ResourceTags.php

```diff
@@ -232,15 +232,12 @@ public function getStatistics($extended = false)
      *
      * @return void
      */
-    public function destroyLinks($resource, $user, $list = null, $tag = null)
+    public function destroyResourceLinks($resource, $user, $list = null, $tag = null)
```

Comment:
Can you add back a deprecated `destroyLinks` method for backward compatibility? It can simply proxy `destroyResourceLinks`, and we can remove it in the next major release.